src/django_htmx/management/commands/downloadhtmx.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING
+from typing import Any
+from urllib.parse import urlparse
+from urllib.parse import urlunparse
+from urllib.request import urlopen
+
+from django.conf import settings
+from django.core.checks import Tags
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
```

Comment:
Only use `if TYPE_CHECKING` to avoid circular imports or other issues, no need for plain imports

```suggestion
from argparse import ArgumentParser
```

Reply:
Done in a91221bab0110ea6d4b09ad5fbd2f0e4e09c2fe2